<!-- %Tag(FULL)%-->
<launch>

 

 <!-- Axes -->


  <node pkg="turtle_teleop" type="turtle_teleop_joy" name="teleop">
    <param name="left_trigger" value="4" type="int"/>
    <param name="right_trigger" value="5" type="int"/>
    <param name="left_y_stick" value="1" type="int"/>
    <param name="right_y_stick" value="3" type="int"/>
  </node>

</launch>
<!-- %EndTag(FULL)%-->
